city_scrapers/spiders/losca_Homeless_Services.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import (
+    BOARD,
+    CITY_COUNCIL,
+    COMMISSION,
+    COMMITTEE,
+    NOT_CLASSIFIED,
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaHomelessServicesSpider(CityScrapersSpider):
+    name = "losca_Homeless_Services"
+    agency = "Los Angeles Homeless Services Authority"
+    timezone = "America/Los_Angeles"
+    start_urls = ["https://www.lahsa.org/events"]
+
+    def parse(self, response):
+        """
+        Parse upcoming meetings for LAHSA.
+        """
+        for item in response.css(".col-lg-8 .list-group a"):
+            # parse classification first
+            # we need it for location
+            classification = self._parse_classification(item)
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=classification,
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=False,
+                time_notes="Start time in Event Link",
+                location=self._parse_location(classification),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
```

Comment:
You can do the same here, and remove the `_parse_source` method on line 112.
```suggestion
                source=response.url,
```